doc: prepare new release 3.8.0 (#4585)
tracim/tracim: changelog.md

# 3.8.0 / 2021-04-27

### New Features

> - Content can be added to personal favourites

Reviewer: ```suggestion
- Contents can be added to personal favourites
```

Author: Done

> - Add and comment a publication

Reviewer: Is this feature the publications by themselves? Publications were added in the 3.7.0 I think.

Author: thats right. its a mistake in project 3.8.0 description.
I remove this part.

Author: done

> - Content can be added to personnal favourites
> - Add and comment a publication
> - Home/login page can be customizable 
> - Preview of links can be genearated in comment
> - Be able to fold the subspaces in the sidebar
> - New space dashboard
> - Possibilities to modify comments
> - New navigation links to user profile
> - Add button "open the gallery" on the content page
> - Allow to open file uploaded in a comment as a content
> - Rich text available in dashboard description

Reviewer 2: ```suggestion
- Content can be added to personal favourites
- Add and comment a publication
- The Home/login page can be customized 
- Links in comments are now previewed when possible
- Subspaces can be folded in the sidebar
- the dashboard has been revamped
- Comments can be edited and removed
- User profile can now be reached from places where the user has posted something 
- There is a new button on the content page to open the gallery
- files uploaded in a comment can now be opened as a content
- The space description is now editable using a rich text editor
```

Reviewer 3: @raphj I update your suggestion with some details (to not have multiples suggestions for the same thing)

Reviewer 3: @raphj I update your suggestion with some details (to not have multiples suggestions for the same thing)

Reviewer 2: good call! :-)

Author: done

### Fixed Issues

> [#](https://github.com/tracim/tracim/issues/)

Reviewer 3: is this "generic" line normal?

Author: no :(

Author: Done

Reviewer 3: ok, so I little suggestion to apply :smile: 
```suggestion
```

Frontend: [#3787](https://github.com/tracim/tracim/issues/3787),
[#3811](https://github.com/tracim/tracim/issues/3811),
[#4464](https://github.com/tracim/tracim/issues/4464),
[#4532](https://github.com/tracim/tracim/issues/4532),
[#4567](https://github.com/tracim/tracim/issues/4567)
Backend: [#4552](https://github.com/tracim/tracim/issues/4552)

### Breaking/Important change

> - Renaming some wording in Tracim (issue [#4348](https://github.com/tracim/tracim/issues/4348)):

Reviewer 2: ```suggestion
- Some rewording in Tracim (issue [#4348](https://github.com/tracim/tracim/issues/4348)):
```

Author: Done
